pkgs/development/tools/continuous-integration/woodpecker/common.nix

```diff
@@ -26,7 +25,7 @@ in
   ldflags = [
     "-s"
     "-w"
-    "-X github.com/woodpecker-ci/woodpecker/version.Version=${version}"
+    "-X go.woodpecker-ci.org/woodpecker/version.Version=${version}"
```

Comment:
It looks like the version isn't picked up correctly, my server thinks I'm running the dev version.